plugins/jdi-elements-plugin/src/main/java/ru/sbtqa/tag/pagefactory/JDIPage.java

```diff
@@ -3,10 +3,9 @@
 import ru.sbtqa.tag.pagefactory.jdi.utils.JDIUtils;
 
 /**
- * Contains basic ru.sbtqa.tag.pagefactory.mobile.actions in particular with jdi elements
- * If we want to extend this functional - inherit from this class
+ * Inherit your jdi page objects from this class
  */
-public abstract class JDIPage extends DefaultPage {
+public abstract class JDIPage implements Page {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Convert the abstract class "JDIPage" into an interface. ([squid:S1610](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1610))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=96&issues=AWZh47E7rFHbA8-EgpnB&open=AWZh47E7rFHbA8-EgpnB)

[ISSUE_KEY]: AWZh47E7rFHbA8-EgpnB